draft-ietf-quic-qlog-main-schema.md

```diff
@@ -891,6 +892,24 @@ instead of logging the "group_id" field with an identical value for each event
 instance, this field is typically logged once in "common_fields", see
 {{common-fields}}.
 
+### system_info
+
+The "sys_info" field can be used to record system-specific details related to an
+event. This is useful, for instance, where an application splits work across
+CPUs, processes, or threads and events for a single trace occur on potentially
+different combinations thereof. Each field is optional to support deployment
+diversity.
+
+Definition:
+
+~~~ cddl
+SystemInformation = {
+  ? processor_id: uint32
+  ? process_id: uint32
+  ? thread_id: uint32
```

Comment:
This only works if the QUIC stack is single-threaded. Is that a reasonable assumption?

Reply:
How so? From the discussion on #152, it seemed fine to have events in the same logical trace that had different thread IDs?

If it seems I'm preventing that in this propsal, I'd like to fix it up so all suggestions welcome.

Reply:
I agree with Lucas here, as also discussed on the original issue.